docs/dev-quickstart.md

```diff
@@ -67,7 +71,7 @@ cert-manager-webhook-6b57b9b886-5cbnp      1/1     Running   0          32s
 As a cluster API~(CAPI) provider, CAPN requires core components of CAPI to be setup. We need to deploy the unreleased version of CAPI for `v1alpha4` API support.
 
 ```console
-git clone git@github.com:kubernetes-sigs/cluster-api.git
+git clone https://github.com/kubernetes-sigs/cluster-api.git
```

Comment:
no need to switch to https here.

Reply:
actually, not everyone use git (ssh) ... but I can leave it as it is